test/lambda.py

```diff
@@ -0,0 +1,22 @@
+import boto3
+import botocore
+import json
+import os
+
+s3 = boto3.resource('s3', region_name = 'us-east-1')
+
+bucket_name = 'inbox-bucket-test'
+bucket_name1 = 'inbox-bucket-testresized'
+
+bucket = s3.Bucket(bucket_name)
+print("created source bucket\n")
+
+bucket1 = s3.Bucket(bucket_name1)
+print("created destination bucket\n")
+
+fileToBeUploaded = os.getcwd()+'/eml/email_sample.eml'
```

Comment:
You can give the filename, unique `from` address to the script `emailgen.py` and then upload the generated blob to the S3 bucket? So that you can do the assertion on the destination bucket quickly.